examples/example-accordionpanel/style/index.css

```diff
@@ -1,8 +1,8 @@
-/* 
+/*
  Copyright (c) Jupyter Development Team.
  Distributed under the terms of the Modified BSD License.
 */
-@import '~@lumino/default-theme/style/index.css';
+@import '@lumino/default-theme/style/index.css';
```

Comment:
I don't really know what I'm doing when it comes to CSS loaders, but after just trying a bunch of different [CSS Rollup plugins](https://github.com/rollup/awesome#css) and various code changes I found that the following worked: 

1. Use `rollup-plugin-styles` 
2. Change any CSS `@import` in the chain to use `@` rather than `~@` syntax.

Reply:
Makes sense to me